content/faq/bounties.md

```diff
@@ -15,12 +15,12 @@ To claim a bounty, please email [bounty@lbry.io](mailto:bounty@lbry.io). Please
 
 ### How do I get compensated?
 
-People who complete bounties are awarded LBRY credits. These credits will be transferred to a wallet address of your choice and are yours with no restrictions to do as you like.
+People who complete bounties are rewarded LBRY credits (LBC). These credits will be transferred to an LBRY wallet address of your choice and are yours, with no restrictions to do as you like.
```

Comment:
I think people just pronounce LBRY "library".  So I wouldn't use 'an'.

Reply:
A lot of people surprisingly don't know (or don't read "LBRY" as library) rather read each individual letter so I just thought an would be appropriate prior to the "L".

Reply:
Amended.